index.html

```diff
@@ -167,7 +167,7 @@ <h3 class="main-title3">Remaining</h3>
                     <div class="event-info-col">
                         <div class="event-info-ico"><span class="fa fa-microphone"></span></div>
                         <h3 class="main-title3">Speakers</h3>
-                        <p><strong>28</strong> Professional Speakers</p>
+                        <p><strong>28</strong>Speakers</p>
```

Comment:
I am not too sure about using the term "Professional Speakers". I think we are trying to express they are python professional and the term does not seem to be conveying the same sentiment